exporters/otlp/include/opentelemetry/exporters/otlp/otlp_http_client.h

```diff
@@ -38,6 +38,8 @@ constexpr char kDefaultMetricsPath[] = "/v1/metrics";
 // The HTTP header "Content-Type"
 constexpr char kHttpJsonContentType[]   = "application/json";
 constexpr char kHttpBinaryContentType[] = "application/x-protobuf";
+// The HTTP header "User-Agent"
+constexpr char kHttpUserAgent[] = "OTel OTLP Exporter Cpp/" OPENTELEMETRY_VERSION;
```

Comment:
nit - should we instead use OPENTELEMETRY_SDK_VERSION? While both refer to same sem-ver value for now, in future if they deviate sdk version would be more relevant.

Reply:
Done